pmd-java/src/main/resources/rulesets/java/strictexception.xml

```diff
@@ -243,6 +243,31 @@ public class Foo extends Error { }
     ]]></example>
   </rule>
 
+  <rule name="DoNotExtendJavaLangThrowable"
+        language="java"
+        since="6.0"
+        message="Exceptions should not extend java.lang.Throwable"
+        class="net.sourceforge.pmd.lang.rule.XPathRule"
+        externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_strictexception.html#donotextendjavalangthrowable">
+    <description>
+Extend Exception or RuntimeException instead of Throwable.
+    </description>
+    <priority>3</priority>
+    <properties>
+        <property name="xpath">
+            <value>
+<![CDATA[
+//ClassOrInterfaceDeclaration/ExtendsList/ClassOrInterfaceType
+  [@Image="Throwable" or @Image="java.lang.Throwable"]
```

Comment:
This is interesting... this would normally require type-resolution, but doing so would detect subclasses of `Throwable` as violations too... but without it, we may have false posisitves when shadowing `Throwable`. I guess we can leave this as i for the time being, but I'm adding an issue to add support for exact type matches on type resolution from XML.